app/mailers/sign_up_support_mailer.rb

```diff
@@ -0,0 +1,14 @@
+class SignUpSupportMailer < MauMailer
+  def secret_word_request(feedback)
+    emails = 'feedback@missionartists.org', 'trish@trishtunney.com' # just in case
```

Comment:
this backup is doing nothing.  cuz `mailer_list` is using `instance` which means it'll always be present.   if we cared, we'd check formatted_emails list but maybe nothing is fine too.